instrumentation/grape/lib/opentelemetry/instrumentation/grape/version.rb

```diff
@@ -8,7 +8,7 @@ module OpenTelemetry
   module Instrumentation
     module Grape
       # Current gem version
-      VERSION = '0.1.1'
+      VERSION = '0.1.2'
```

Comment:
```suggestion
      VERSION = '0.1.1'
```

Our release process automatically bumps versions. Please revert this.

Reply:
oh my bad, reverted the change ✅